docs/user-guide/scaling.rst

```diff
@@ -276,6 +278,22 @@ remote logging and aggregation, refer to the documentation for the log daemon
 running on your system.
 
 
+.. _clustered_monitoring:
+
+Cluster Monitoring
+^^^^^^^^^^^^^^^^^^
+
+A clustered deployment can be monitored with the techniques described in
+:ref:`getting_the_server_status`.
+
+.. warning:: Information provided by the ``/status/`` API call does not
+    include ``httpd`` status information. It is recommended that each
+    `Pulp clustered server` acting as a webserver have its ``/status/``
+    API queried directly and not through the load balancer which may
```

Comment:
This sentence turned into a run-on. How about something like this? "... queried directly, not through the load balancer, to avoid unexpected request routing."